docs/antctl.md

```diff
@@ -0,0 +1,53 @@
+# Antctl
+
+Antctl is the command-line tool for Antrea. At the moment, antctl supports
+running in two different modes:
+ * "controller mode": when run out-of-cluster or from within the Antrea
+ Controller Pod, antctl can connect to the Antrea Controller and query
+ information from it (e.g. the set of computed network policies).
+ * "agent mode": when run from within an Antrea Agent Pod, antctl can connect to
+ the Antrea Agent and query information local to that Agent (e.g. the set of
+ computed network policies received by that Agent from the Antrea Controller, as
+ opposed to the entire set of computed policies).
+
+## Installation
+
+The antctl binary is included in the Antrea Docker image
+(`antrea/antrea-ubuntu`) which means that there is no need to install anything
+to connect to the Antrea Agent. Simply exec into the antrea-agent container for
+the appropriate antrea-agent Pod and run `antctl`:
+```
+kubectl exec -it <antrea-agent Pod name> -n kube-system -c antrea-agent bash
+> antctl help
+```
+
+Starting with Antrea release v0.5.0, we publish the antctl binaries for
+different OS / CPU Architecture combinations. Head to the [releases
+page](https://github.com/vmware-tanzu/antrea/releases) and download the
+appropriate one for your machine. For example:
+
+On Mac & Linux:
+```
+curl -Lo ./antctl "https://github.com/vmware-tanzu/antrea/releases/download/v0.5.0/antctl-$(uname)-x86_64"
```

Comment:
Will we support different arch also?

Reply:
Yes, see https://github.com/vmware-tanzu/antrea/pull/517
I added one more sentence below to explicitly mention it

Reply:
Got it, I used to worried about arm arch compiling.